src/main/java/ro/societateahermes/backendservice/service/serviceImplementation/FormService.java

```diff
@@ -0,0 +1,38 @@
+package ro.societateahermes.backendservice.service.serviceImplementation;
+
+import org.springframework.stereotype.Service;
+import ro.societateahermes.backendservice.entities.form.Form;
+import ro.societateahermes.backendservice.entities.form.Option;
+import ro.societateahermes.backendservice.entities.form.Question;
+import ro.societateahermes.backendservice.repository.FormRepository;
+
+import java.util.List;
+
+@Service
+public class FormService {
```

Comment:
Serviceul trebuie sa implementeze o interfata. In felul asta ne va fi usor daca candva in viitor vom vrea sa-l schimbam pentru ca va trebui doar sa inlocuim fisierul .java. Daca lasam fara interfata va trebui sa trecem prin fiecare fisier in care declaram serviceul si sa modificam manual declaratiile